steps/install-bundler/1.0.1/step.yml

```diff
@@ -0,0 +1,31 @@
+title: Install specific bundler version
+summary: This step allows you to uninstall the system bundler version, and match the
+  one in your Gemfile.
+description: This step allows you to uninstall the system bundler version, and match
+  the one in your Gemfile..
+website: https://github.com/FutureWorkshops/bitrise-step-install-bundler
+source_code_url: https://github.com/FutureWorkshops/bitrise-step-install-bundler
+published_at: 2019-02-06T18:58:36.415567+01:00
+source:
+  git: https://github.com/FutureWorkshops/bitrise-step-install-bundler.git
+  commit: a85500a815476ed06008d301563cddcb1ca4ad7d
+host_os_tags:
+- osx-10.10
+- osx-10.9
+project_type_tags:
+- ios
+- xamarin
+type_tags:
+- xcode
```

Comment:
The CI failed because of a wrong type_tag, please read our Step Development guideline to see which project_type_tags and type_tags can be used: https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#step-grouping-convention